src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -214,10 +213,11 @@ public class InstancesController : ControllerBase
                     string selfUrl = $"{host}{url}{selfQueryString}";
 
                     response.Self = selfUrl;
-                }
+                }               
 
-                if (nextContinuationToken != null)
+                if (!string.IsNullOrEmpty(result.ContinuationToken))
```

Comment:
result.ContinuationToken is being set to null on line 192

Reply:
Changed the if check now.